server/src/infra/repositories/media.repository.ts

```diff
@@ -1,8 +1,10 @@
 import { CropOptions, IMediaRepository, ResizeOptions, TranscodeOptions, VideoInfo } from '@app/domain';
+import { Colorspace } from '@app/infra/entities';
 import { Logger } from '@nestjs/common';
 import ffmpeg, { FfprobeData } from 'fluent-ffmpeg';
 import fs from 'fs/promises';
 import sharp from 'sharp';
+import { Writable } from 'typeorm/platform/PlatformTools.js';
```

Comment:
This is probably the wrong import, no?

Reply:
Yeah, not sure why it auto-imported this one